ledger-api/grpc-definitions/com/daml/ledger/api/v1/update_service.proto

```diff
@@ -48,15 +49,16 @@ service TransactionService {
   // - ``INVALID_ARGUMENT``: if the payload is malformed or is missing required fields (e.g. if requesting parties are invalid or empty)
   rpc GetTransactionByEventId (GetTransactionByEventIdRequest) returns (GetTransactionResponse);
 
-  // Lookup a transaction tree by its ID.
-  // For looking up a transaction instead of a transaction tree, please see GetFlatTransactionById
+  // Lookup a transaction tree or a transfer by its ID.
```

Comment:
Rather than spelling out `transaction or transfer` everywhere, I'd recommend to define at a central place what form an `Update` can take (currently transaction or transfer) and then use the `update` term everywhere.